src/ray/protobuf/gcs_service.proto

```diff
@@ -513,6 +513,52 @@ service PlacementGroupInfoGcsService {
   rpc WaitPlacementGroupUntilReady(WaitPlacementGroupUntilReadyRequest)
       returns (WaitPlacementGroupUntilReadyReply);
 }
+
+message GetRequest {
```

Comment:
I think we need to move this into the autoscaler backwards compatibility section unless we find a way to not break the autoscaler.